### PR TITLE
[xdoctest] reformat example code with google style in No.150-160

### PR DESCRIPTION
**Pull request by enkilee**
### PR types

Others
### PR changes

Others
### Description

150  | python/paddle/text/datasets/wmt16.py
151  | python/paddle/text/viterbi_decode.py
152  | python/paddle/vision/models/_utils.py
153  | python/paddle/vision/models/alexnet.py
154  | python/paddle/vision/models/densenet.py 
155  | python/paddle/vision/models/googlenet.py
156  | python/paddle/vision/models/inceptionv3.py
157  | python/paddle/vision/models/lenet.py
158  | python/paddle/vision/models/mobilenetv1.py
159  | python/paddle/vision/models/mobilenetv2.py
160  | python/paddle/vision/models/mobilenetv3.py

预览：

en: http://preview-paddle-pr-56178.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/en/api/paddle/abs_en.html
zh: http://preview-paddle-pr-56178.paddle-docs-preview.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/abs_cn.html

关联 PR：
https://github.com/PaddlePaddle/Paddle/issues/55629
https://github.com/PaddlePaddle/Paddle/pull/55295

@sunzhongkai588 @SigureMo @megemini

### CODE REVIEW

File: python/paddle/text/datasets/wmt16.py [hunk 1]
```diff
@@ -77,30 +77,39 @@ class WMT16(Dataset):
...
+            [89] [32] [33]
+            [79] [18] [19]
+            [55] [26] [27]
+            [147] [36] [37]
+            [106] [22] [23]
+            [135] [50] [51]
+            [54] [43] [44]
+            [217] [30] [31]
+            [146] [51] [52]
+            [55] [24] [25]
```
Comment (SigureMo) [inline python/paddle/text/datasets/wmt16.py]: image

0D 支持后，shape 应该不再是 `[1]` 了，可以用最新的 develop 试一下

File: python/paddle/text/datasets/wmt16.py [hunk 2]
```diff
@@ -77,30 +77,39 @@ class WMT16(Dataset):
...
+            >>> from paddle.text.datasets import WMT16
+
+            >>> class SimpleNet(paddle.nn.Layer):
+            ...     def __init__(self):
+            ...         super().__init__()
+            ...
+            ...     def forward(self, src_ids, trg_ids, trg_ids_next):
+            ...         return paddle.sum(src_ids), paddle.sum(trg_ids), paddle.sum(trg_ids_next)
+            ...
+            >>> paddle.disable_static()
```
Comment (SigureMo) [inline python/paddle/text/datasets/wmt16.py]: 默认动态图应该不需要这一行了

File: python/paddle/text/viterbi_decode.py [hunk 3]
```diff
@@ -104,27 +111,34 @@ class ViterbiDecoder(Layer):
...
+            >>> transition = paddle.rand((num_tags, num_tags), dtype='float32')
+            >>> decoder = paddle.text.ViterbiDecoder(transition, include_bos_eos_tag=False)
+            >>> scores, path = decoder(emission, length)
+            >>> print(scores)
+            Tensor(shape=[2], dtype=float32, place=Place(cpu), stop_gradient=True,
+            [2.57385254, 2.04533720])
+            >>> print(path)
+            Tensor(shape=[2, 2], dtype=int64, place=Place(cpu), stop_gradient=True,
+            [[0, 0],
+              [1, 1]])
```
Comment (SigureMo) [inline python/paddle/text/viterbi_decode.py]:
```suggestion
            [[0, 0],
             [1, 1]])
```

File: python/paddle/vision/models/_utils.py [hunk 4]
```diff
@@ -50,22 +50,24 @@ class IntermediateLayerGetter(nn.LayerDict):
...
-        print([(k, v.shape) for k, v in out.items()])
-        # [('feat1', [1, 64, 56, 56]), ('feat2', [1, 256, 14, 14])]
+        >>> import paddle
+        >>> m = paddle.vision.models.resnet18(pretrained=False)
+        >>> # extract layer1 and layer3, giving as names `feat1` and feat2`
+        >>> new_m = paddle.vision.models._utils.IntermediateLayerGetter(m,
+        ...     {'layer1': 'feat1', 'layer3': 'feat2'})
+        >>> out = new_m(paddle.rand([1, 3, 224, 224]))
+        >>> print([(k, v.shape) for k, v in out.items()])
+        [('feat1', [1, 64, 56, 56]), ('feat2', [1, 256, 14, 14])]
```
Comment (SigureMo) [inline python/paddle/vision/models/_utils.py]: 整体缩进层级 +1

File: python/paddle/vision/models/mobilenetv2.py [hunk 5]
```diff
@@ -217,23 +216,23 @@ def mobilenet_v2(pretrained=False, scale=1.0, **kwargs):
...
+            >>> model = mobilenet_v2()
 
-            # build model and load imagenet pretrained weight
-            # model = mobilenet_v2(pretrained=True)
+            >>> # Build model and load imagenet pretrained weight
+            >>> # model = mobilenet_v2(pretrained=True)
 
-            # build mobilenet v2 with scale=0.5
+            >>> # Build mobilenet v2 with scale=0.5
             model = mobilenet_v2(scale=0.5)
```
Comment (SigureMo) [inline python/paddle/vision/models/mobilenetv2.py]: 这咋跑输出行了

File: python/paddle/text/datasets/wmt16.py (hunk 1 above)
Comment (enkilee) [reply python/paddle/text/datasets/wmt16.py]: 收到！

File: python/paddle/text/datasets/wmt16.py (hunk 2 above)
Comment (enkilee) [reply python/paddle/text/datasets/wmt16.py]: 收到

File: python/paddle/text/viterbi_decode.py (hunk 3 above)
Comment (enkilee) [reply python/paddle/text/viterbi_decode.py]: 收到

File: python/paddle/vision/models/_utils.py (hunk 4 above)
Comment (enkilee) [reply python/paddle/vision/models/_utils.py]: 收到

File: python/paddle/vision/models/mobilenetv2.py (hunk 5 above)
Comment (enkilee) [reply python/paddle/vision/models/mobilenetv2.py]: 收到

**Review by megemini — APPROVED** [review]
LGTM ～ 赞～

**Review by SigureMo — APPROVED** [review]
LGTMeow 🐾

**Pull request merged**